twitter/search.go

```diff
@@ -49,6 +49,7 @@ type SearchTweetParams struct {
 	MaxID           int64  `url:"max_id,omitempty"`
 	Until           string `url:"until,omitempty"`
 	IncludeEntities *bool  `url:"include_entities,omitempty"`
+	TweetMode       string `url:"tweet_mode,omitempty"`
```

Comment:
Where can a user of this Go package find out what string values `TweetMode` field can be set to?

Is it available in Twitter API docs?